### ISSUE TITLE
Force qname to use lowercase

### ISSUE THREAD

**Issue by KVInventoR** [dmachard/dnstap-receiver#15]
In some cases I see queries with different cases to dns domain, it's difficult to build statistics when 1 domain has different cases:
Google.Com
google.COM
GOOLE.COM

Can you please implement some feature which will give possibility to convert all domains statistics to use lower case for all domains?

**Comment by dmachard** [comment]
new parameter qname-lowercase added in config file (section statistics)

```
# dnstap receiver statistics
statistics:
  # qname lowercase
  qname-lowercase: true
```

**Comment by KVInventoR** [comment]
thank you,
I will test it

**Comment by dmachard** [comment]
implemented in release 3.0.0
